src/pages/NewAsset.vue

```diff
@@ -122,6 +123,41 @@ export default {
       const timeUnit = get(this.selectedAssetType, 'timing.timeUnit')
       return this.$t({ id: 'pricing.price_per_time_unit_label' }, { timeUnit })
     },
+    step () {
```

Comment:
For consistency I suggest you just try and remove this check in `createAsset` method to fix the button:

https://github.com/abruere/workingoo/blob/5e2cc2f38be654b56cdd9813fbe1b4896b2df2f1/src/pages/NewAsset.vue#L234

Reply:
I don't really get it. Should I remove all function `createAsset` or just the line `if (this.step > 3) { ` ?

Reply:
Just the `if` check, line 234 shown above + this line closing the block:

https://github.com/abruere/workingoo/blob/5e2cc2f38be654b56cdd9813fbe1b4896b2df2f1/src/pages/NewAsset.vue#L357

(And unindent the block)

This is what caused the `createAsset` button to have no effect after your PR. You removed `step` computed function but not this check.

I’m just suggesting you remove the check rather than re-introducing `step` function you said you didn’t need anymore.

Reply:
Okay. I did it now. 
Thank you